app/views/posts/index.html.erb

```diff
@@ -33,16 +33,28 @@
                 </div>
             </div>
 
+         <% if can? :destroy, @post %>
+          <%= form_with url: user_post_path(@user.id, post.id), method: :delete do |f| %>
+            <%= f.submit 'Delete post', class: 'btn-danger btn-sm mx-2' %>
+          <% end %>
+        <% end %>
```

Comment:
Well done using the `can?` method for all your delete buttons!!! 💯 👏🏾

Reply:
thank you